helm/fiftyone-teams-app/values.yaml

```diff
@@ -323,6 +323,10 @@ delegatedOperatorExecutorSettings:
   enabled: false
   # Environment Variables are passed to the delegated-operator-executor containers.
   env:
+    # -- Full path to a network-mounted file system or a cloud storage path to
+    # use for storing logs generated by delegated operation runs, one file
+    # per job. The default `null` means log upload is disabled.
+    FIFTYONE_DELEGATED_OPERATION_RUN_LINK_PATH: null
```

Comment:
This is a nitpick, so disregard if you want, but this would probably be better as an empty string, `""`, so the type is defined.

Reply:
Yeah I get that. The thing I don't like is specifying default value in 2 places. If we change the default in code, we have to remember to change it here too or else it would work when running locally but not on any deployed environment.

But that is a larger configuration consolidation project I'd love to do but will never have time for lol.

In meantime sure I changed the consumer of this variable to treat unset and empty string the same way, so I can have it be empty string now.

Reply:
In the context of helm, both `null` and `""` will interpreted as the same thing to the pod. They will both be treated as:

```shell
FIFTYONE_DELEGATED_OPERATION_RUN_LINK_PATH=
```

from inside the container.

Reply:
Hmm that is weird because the unit tests behave like you're saying but when I do `helm template` it completely omits `FIFTYONE_DELEGATED_OPERATION_RUN_LINK_PATH` from the env.

Reply:
When I run:

```shell
helm template . --set delegatedOperatorExecutorSettings.enabled=true > out
```

I see:

```yaml
            - name: FIFTYONE_ENCRYPTION_KEY
              valueFrom:
                secretKeyRef:
                  name: fiftyone-teams-secrets
                  key: encryptionKey
            - name: FIFTYONE_DELEGATED_OPERATION_RUN_LINK_PATH <------
              value: 
            - name: FIFTYONE_INTERNAL_SERVICE
              value: "true"
            - name: FIFTYONE_MEDIA_CACHE_SIZE_BYTES
              value: "-1"
```

Reply:
maybe you're missing the `--set delegatedOperatorExecutorSettings.enabled=true`?

Reply:
Oh, yes, I see. Yeah, values get evaluated from left to right... so if there's a value that matches multiple files, it'll just use whatever was passed in last (the rightmost). Maybe something funky in there. TLDR; though, PR is approved on my end! 👍